package.json

```diff
@@ -48,9 +48,9 @@
     "aframe": "^1.4.0",
     "aframe-event-set-component": "^5.0.0",
     "aframe-extras": "^6.1.1",
-    "aframe-machinima-testing": "^0.1.2",
+    "aframe-machinima-testing": "^0.0.1",
     "aframe-motion-capture-components": "github:wmurphyrd/aframe-motion-capture-components#v0.2.8a",
-    "babel-preset-env": "^1.6.0",
+    "babel-preset-env": "^0.0.0",
```

Comment:
It doesn't make sense to downgrade those dependencies. If you used `npm audit fix` it can downgrade to versions that didn't have the security vulnerability. But that's not the solution here.